mismi-s3/main/s3.hs

```diff
@@ -88,9 +88,8 @@ run = \case
 
 runK :: AmazonkaCommand -> IO ()
 runK k = do
-  mr <- getRegionFromEnv
-  let r = fromMaybe Sydney mr
-  orDie renderError . EitherT . tryJust exceptExit . runAWSWithRegion r $ case k of
+  e <- orDie envErrorRender . EitherT $ discoverAwsEnv
```

Comment:
I think this plays into the conversation about having a better/more-consistent env-var based config library.